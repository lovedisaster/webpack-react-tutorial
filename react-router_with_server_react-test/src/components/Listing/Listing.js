import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';

class Listing extends PureComponent {
    render() {
        return (
            <div>
                <h1>This is the listing page</h1>
            </div>
        );
    }
}

Listing.propTypes = {

};

export default Listing;