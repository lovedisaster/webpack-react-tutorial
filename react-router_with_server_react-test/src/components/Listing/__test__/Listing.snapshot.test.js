import React from 'react';
import Listing from '../Listing';
import TestRenderer from 'react-test-renderer';

it("listing page snapshot" , () => {
    const listingPage = TestRenderer.create(<Listing></Listing>).toJSON();
    expect(listingPage).toMatchSnapshot();
});