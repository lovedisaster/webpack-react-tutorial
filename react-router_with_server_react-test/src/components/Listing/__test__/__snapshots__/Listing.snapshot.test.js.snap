// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`listing page snapshot 1`] = `
<div>
  <h1>
    This is the listing page
  </h1>
</div>
`;
